roles/python/tasks/install_dependencies.yml

```diff
@@ -42,7 +42,9 @@
   register: _pyproject_file
 
 - name: Install python dependencies from pyproject
-  when: _pyproject_file.stat.exists
+  when:
+    - not _requirements_file.stat.exists
```

Comment:
this is an unrelated change: recently added logic to install python dependencies from a different config file (needed for simulating risk), but when running this discovered that an additional check is needed for when to run that install (geniza has the pyproject file to configure a tool but doesn't use it for python requirements)